package.json

```diff
@@ -39,7 +39,7 @@
     "dataloader": "^1.2.0",
     "debug": "^2.2.0",
     "finalhandler": "^0.5.0",
-    "graphql": ">=0.6.0 <1.0.0",
+    "graphql": ">=0.8.0-beta1",
```

Comment:
Is there a reason we can’t keep the dependency the way it was? The reason we defined such a wide range is so that multiple GraphQL JS instances would not conflict.